src/main.rs

```diff
@@ -261,6 +262,11 @@ fn create_app<'a, 'b>(
                 .conflicts_with_all(&["net-backend", "kernel"])
                 .min_values(1),
         )
+        .arg(
+            Arg::with_name("disable-seccomp")
```

Comment:
Maybe this should be `--secomp=<true|false>` it could then be extended to just make it log which would help with debugging further.

Reply:
Sorry I don't get the relation with the logging.
Also, I like how simple `disable-seccomp` is. We have a lot of options with `on|off`, but in this case we can make things even simpler.

Reply:
`--seccomp=log` would pass `SECCOMP_FILTER_FLAG_LOG` so that the denied entries would be logged making it easy to understand how the filter needs to be extended.

I think the boolean is too limiting and i'm sure we will want to add variants (maybe one which locks all file access down to those at boot.) It would be easier to use the more flexible option now rather than having to change it and break our "command line API" in the future.

Reply:
Ok let me do that then.